common/src/main/java/co/sodalabs/view/CanvasExtensions.kt

```diff
@@ -0,0 +1,11 @@
+package co.sodalabs.view
+
+import android.graphics.Canvas
+
+fun Canvas.runSafely(
```

Comment:
Wow, nice extension

Reply:
We could make it another separate private module.